src/ui/types/index.ts

```diff
@@ -144,6 +144,7 @@ export interface Recording {
   operations?: OperationsData;
   resolution: { resolvedAt: string; resolvedBy: string };
   collaboratorRequests: CollaboratorRequest[];
+  metadata: any;
```

Comment:
Would be good to tighten this up. It's something like `Record<string, null | Record<string, unknown>>` which is a bit weird.